nixos/modules/services/misc/ombi.nix

```diff
@@ -0,0 +1,76 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let cfg = config.services.ombi;
+
+in {
+  options = {
+    services.ombi = {
+      enable = mkEnableOption "Ombi";
```

Comment:
Maybe you want to include a link to https://docs.ombi.app/info/reverse-proxy/ 🤷‍♂️

Reply:
I added a note about reverse proxies, please reopen if I did wrong or if you have more suggestions.